.env.example

```
@@ -1,7 +1,14 @@
 DATABASE_URL="postgresql://postgres:password@localhost:5432/db-name"
 BETTER_AUTH_SECRET="change-me-generate-with-openssl-rand-base64-32"
 BETTER_AUTH_URL="http://localhost:3001"
+AWS_ACCESS_KEY_ID=""
```

dotenv-linter (4.0.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)